operator/pkg/manifest/installer.go

```diff
@@ -495,39 +495,55 @@ func buildComponentApplyOutput(stdout string, stderr string, objects object.K8sO
 	}
 }
 
+func istioCustomResourceDefinition(group string) bool {
+	switch group {
+	case "config.istio.io",
+		"rbac.istio.io",
+		"security.istio.io",
+		"authentication.istio.io",
+		"networking.istio.io":
+		return true
+	}
+	return false
+}
+
 // DefaultObjectOrder is default sorting function used to sort k8s objects.
 func DefaultObjectOrder() func(o *object.K8sObject) int {
 	return func(o *object.K8sObject) int {
 		gk := o.Group + "/" + o.Kind
-		switch gk {
+		switch {
 		// Create CRDs asap - both because they are slow and because we will likely create instances of them soon
-		case "apiextensions.k8s.io/CustomResourceDefinition":
+		case gk == "apiextensions.k8s.io/CustomResourceDefinition":
 			return -1000
 
 			// We need to create ServiceAccounts, Roles before we bind them with a RoleBinding
-		case "/ServiceAccount", "rbac.authorization.k8s.io/ClusterRole":
+		case gk == "/ServiceAccount" || gk == "rbac.authorization.k8s.io/ClusterRole":
 			return 1
-		case "rbac.authorization.k8s.io/ClusterRoleBinding":
+		case gk == "rbac.authorization.k8s.io/ClusterRoleBinding":
 			return 2
 
-			// Validation webook maybe impact CRs applied later
-		case "admissionregistration.k8s.io/ValidatingWebhookConfiguration":
+			// install Istio API before we apply the validatingwebhookconfiguration
+		case istioCustomResourceDefinition(o.Group):
```

Comment:
note to self: swap case 3 and 4 here so the webhook configuration is always installed first.